src/exception_page/frame.cr

```diff
@@ -49,12 +49,10 @@ struct ExceptionPage::Frame
 
   def label : String
     case file
-    when .includes?("/crystal/")
+    when .includes?("/crystal/"), .includes?("/crystal-lang/")
```

Comment:
Nice refactor! I didn't know you could do that 👍

Reply:
Sometimes while coding in Ruby I'm trying to do the same, and every time I'm surprised how expressive Crystal's syntactic sugar can be - and Ruby surprisingly ain't ;) Love it more and more with every year...